code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1356,8 +1356,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 					alert("ERROR: Incorrect / improper path given.")
 					return
 			//send the pod
-			target.Stun(10)//takes 0.53 seconds for CentCom pod to land
-			new /obj/effect/DPtarget(get_turf(target), delivery, POD_CENTCOM, target)
+			if(!isobserver(target))

```

Comment:
This should be istype check at the start not the other way around.

Reply:
What to you mean "other way around"? Is it better practice to put the isobserver check at the top of that block?

Reply:
I mean it shouldn't be checking against a single thing that is excluded, instead check what's allowed and work depending on that.

Reply:
ooh